src/java/picard/sam/CompareSAMs.java

```diff
@@ -50,16 +50,28 @@
  * @author alecw@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "USAGE: CompareSAMS <SAMFile1> <SAMFile2>\n" +
-                "Compares the headers of the two input SAM or BAM files, and, if possible, the SAMRecords. " +
-                "For SAMRecords, compares only the readUnmapped flag, reference name, start position and strand. " +
-                "Reports the number of SAMRecords that match, differ in alignment, are mapped in only one input, " +
-                "or are missing in one of the files",
-        usageShort = "Compares two input SAM or BAM files",
+        usage = CompareSAMs.USAGE_SUMMARY + CompareSAMs.USAGE_DETAILS,
+        usageShort = CompareSAMs.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class CompareSAMs extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Compares two input \".sam\" or \".bam\" files.  ";
+    static final String USAGE_DETAILS = "Starts by comparing the headers of \".sam\" or \".bam\" files and if " +
```

Comment:
This first sentence is a little awkward -- could be confusing to non-native speakers. Should simplify the grammar.